### ISSUE TITLE
[Issue]: streaming crash after 8 minutes

### ISSUE THREAD

**Issue by Zawardo1000** [moonlight-stream/moonlight-android#1323]
### Describe the bug

my issue seems simimilar to 527 on ios.
after more or less 8 minutes the client disconnects. tested on multiple android devices.

### Steps to reproduce

everytime with any client.
host sunshine 0.21 on windows 10 with gtx 980 (no hags possible)
every setting default.
tested on host with wired and wireless

### Affected games

everything

### Other Moonlight clients

PC

### Moonlight adjusted settings

No

### Moonlight adjusted settings (please complete the following information)

tried to lower priority

### Moonlight default settings

Yes

### Gamepad-related connection issue

No

### Gamepad-related input issue

Yes

### Gamepad-related streaming issue

Yes

### Android version

8-12-13

### Device model

odin2, mipad4,oneplus nord2

### Server PC OS version

win 10 22h2

### Server PC GeForce Experience version

3.27.0.120

### Server PC Nvidia GPU driver version

551.23

### Server PC antivirus and firewall software

avast (disabled)

### Relevant log output

```Shell
#0(W)[2024-01-29 18:09:43,090]=19:09:43={6040}<ClientControl>    waitForClientCommand: No packets seen on control channel from 7519 ms, time-out: 500 ms, last command 200 received, dispatch queue size is 18446744073709551615.
#1(W)[2024-01-29 18:09:43,592]=19:09:43={6040}<ClientControl>    waitForClientCommand: No packets seen on control channel from 8021 ms, time-out: 500 ms, last command 200 received, dispatch queue size is 18446744073709551615.
#2(W)[2024-01-29 18:09:44,079]=19:09:44={12344}<EnetProtocol>     Packet 887 on channel 255 retransmitted 27 times; current timeout is 500 and timeout limit is 160
#3(W)[2024-01-29 18:09:44,093]=19:09:44={6040}<ClientControl>    waitForClientCommand: No packets seen on control channel from 8522 ms, time-out: 500 ms, last command 200 received, dispatch queue size is 18446744073709551615.
#4(I)[2024-01-29 18:09:44,350]=19:09:44={2192}<QosManagerTxNati> Frame Number: 30063, Client Q-score: 100, RTD: 0 ms, BW score: 100, Loss score: 100, Latency score: 100, Avg Game FPS: 60.02
#5(W)[2024-01-29 18:09:44,594]=19:09:44={6040}<ClientControl>    waitForClientCommand: No packets seen on control channel from 9023 ms, time-out: 500 ms, last command 200 received, dispatch queue size is 18446744073709551615.
#6(W)[2024-01-29 18:09:45,080]=19:09:45={12344}<EnetProtocol>     Packet 887 on channel 255 retransmitted 29 times; current timeout is 500 and timeout limit is 160
#7(W)[2024-01-29 18:09:45,095]=19:09:45={6040}<ClientControl>    waitForClientCommand: No packets seen on control channel from 9524 ms, time-out: 500 ms, last command 200 received, dispatch queue size is 18446744073709551615.
#8(W)[2024-01-29 18:09:45,597]=19:09:45={6040}<ClientControl>    waitForClientCommand: No packets seen on control channel from 10026 ms, time-out: 500 ms, last command 200 received, dispatch queue size is 18446744073709551615.
#9(M)[2024-01-29 18:09:45,597]=19:09:45={6040}<ClientControl>    Client has not responded in 10026 ms, triggering disconnection
#0(I)[2024-01-29 18:09:45,597]=19:09:45={6040}<ClientControl>    Found 18446744073709551615 packets in socket queue that are not dispatched
#1(D)[2024-01-29 18:09:45,597]=19:09:45={6040}<StopEvent>        Setting event 'Session'. ClientControl:  (0x80040018 NVST_NETERR_CLIENT_TIMED_OUT_PACKETS_IN_DISPATCH_QUEUE)
#2(I)[2024-01-29 18:09:45,597]=19:09:45={2236}<NvscStreamingSes> Stop event received, tearing down.
#3(I)[2024-01-29 18:09:45,597]=19:09:45={2236}<NvscStreamingSes> Starting teardown
#4(I)[2024-01-29 18:09:45,597]=19:09:45={2236}<NvscStreamingSes> Quit requested, waiting on child threads to quit
#5(I)[2024-01-29 18:09:45,597]=19:09:45={2236}<StopEvent>        Termination infos:
#6(I)[2024-01-29 18:09:45,597]=19:09:45={2236}<StopEvent>        - Source 'ClientControl', reason '', code 0x80040018 NVST_NETERR_CLIENT_TIMED_OUT_PACKETS_IN_DISPATCH_QUEUE
#7(I)[2024-01-29 18:09:45,597]=19:09:45={2236}<NvscStreamingSes> Termination reason to report: NVST_NETERR_CLIENT_TIMED_OUT_PACKETS_IN_DISPATCH_QUEUE
#8(I)[2024-01-29 18:09:45,597]=19:09:45={6040}<ClientControl>    Cleaning up after client control thread
#9(D)[2024-01-29 18:09:45,597]=19:09:45={6040}<ClientControl>    Last command received: 200
#0(D)[2024-01-29 18:09:45,597]=19:09:45={2236}<VideoStreamerThr> Destroying video streamer thread.
#1(I)[2024-01-29 18:09:45,597]=19:09:45={6040}<ClientControl>    ClientControl::ThreadFunc() last sent: 1247 ms, last received: 10026 ms, last processed: 10026 ms
#2(D)[2024-01-29 18:09:45,597]=19:09:45={6040}<StopEvent>        Append to 'Session'. ClientControl:  (0x80030000 NVST_DISCONN_INTENDED)
#3(D)[2024-01-29 18:09:45,597]=19:09:45={10516}<NvThread>         About to destroy native thread: VideoStreamerThread (thread id: 0000000000000890)
#4(I)[2024-01-29 18:09:45,597]=19:09:45={17704}<StreamSdkEvents>  Invoking diagnosticsEventRaised TerminationReason
#5(I)[2024-01-29 18:09:45,597]=19:09:45={6040}<ClientControl>    Finished cleaning up after client control thread
#6(D)[2024-01-29 18:09:45,597]=19:09:45={6040}<NvThread>         Completed thread function for native thread: ClientControl (thread id: 0000000000001798)
#7(I)[2024-01-29 18:09:45,597]=19:09:45={17704}<NvStreamServerCo> DiagnosticEvent: Reported UnifiedErrorCode 8043800e for SessionTerminationReason 800e
#8(I)[2024-01-29 18:09:45,597]=19:09:45={17576}<Server>           Server has been asked to end the current streaming session.
#9(I)[2024-01-29 18:09:45,597]=19:09:45={17576}<Server>           Server is now in state: EndingSession
#0(D)[2024-01-29 18:09:45,598]=19:09:45={17576}<StopEvent>        Append to 'Session'. ClientControl:  (0x80040018 NVST_NETERR_CLIENT_TIMED_OUT_PACKETS_IN_DISPATCH_QUEUE)
#1(I)[2024-01-29 18:09:45,598]=19:09:45={17576}<RtspHandshake>    Terminating RTSP session
#2(I)[2024-01-29 18:09:45,598]=19:09:45={17704}<StreamSdkEvents>  Invoking diagnosticsEventRaised EnetDispatchQueued
#3(I)[2024-01-29 18:09:45,598]=19:09:45={17576}<NativeServer>     Waiting for session thread to complete.
#4(I)[2024-01-29 18:09:45,598]=19:09:45={17704}<StreamSdkEvents>  Invoking clientEventCallback NvstEventDetail::NVST_ED_RTSP_TEARDOWN_SUCCESS: 
#5(M)[2024-01-29 18:09:45,598]=19:09:45={17704}<SciBase>          Sent event RtspTeardownSuccess: 
#6(E)[2024-01-29 18:09:45,600]=19:09:45={2192}<VideoStreamer>    Failed to enqueue frame packet for stream:0 frame 30137
#7(M)[2024-01-29 18:09:45,608]=19:09:45={2192}<VideoStreamer>    Perform video streaming is done. stream:0 Actual frames:30136. Frame Number 30137
#8(I)[2024-01-29 18:09:45,608]=19:09:45={2192}<VideoContext>     VideoStreamEventRaised
#9(D)[2024-01-29 18:09:45,608]=19:09:45={4592}<NvThread>         About to destroy native thread: CaptureThread (thread id: 0000000000002E8C)
#0(E)[2024-01-29 18:09:45,608]=19:09:45={11916}<VirtualVsyncServ> nvstVirtualVsyncSetActualStreamingIntervals failed due to StreamSdkException 800b1000 [NVST_R_INVALID_STATE] Cannot run the requested logic for state (1005) is incorrect.
#1(M)[2024-01-29 18:09:45,621]=19:09:45={11916}<CaptureThread>    Finished with status STATUS_RUNNING after capturing 30138 frames
#2(I)[2024-01-29 18:09:45,621]=19:09:45={11916}<CaptureThread>    Capture ended normally
```

**Comment by Zawardo1000** [comment]
forgot to say with moonlight pc client (windows 64bit) everything is fine.
